spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/data/mongo/AutoConfigureEmbeddedTestMongod.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2012-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.test.autoconfigure.data.mongo;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.properties.PropertyMapping;
+
+/**
+ * Annotation that can be applied to a test class to disable the usage of an
+ * embedded Mongod even if
+ * {@code de.flapdoodle.embed:de.flapdoodle.embed.mongo} is on the classpath.
+ *
+ * @author Michael J. Simons
+ * @since 1.5.0
+ * @see EmbeddedTestMongodAutoConfiguration
+ */
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@ImportAutoConfiguration
+@PropertyMapping("spring.test.mongod.embedded")
+public @interface AutoConfigureEmbeddedTestMongod {
```

Comment:
That sounds like a generic name. One would expect a `AutoconfigureTestMongo` here. Even so, I am not sure what would be its purpose.

What is the use case of having flapdoodle on the classpath and not willing to use it? If that's a thing, maybe we should offer a way for the curent auto-configuration to back-off.

If there isn't a compelling use case, I'd rather remove that altogether